Neuron-Startup.lua

```diff
@@ -20,38 +20,38 @@ end
 function Neuron:RegisterBars()
 
 	--Neuron Action Bar
-	Neuron:RegisterBarClass("ActionBar", "ActionBar", L["Action Bar"], "Action Button", DB.ActionBar, Neuron.ACTIONBUTTON, 250)
+	Neuron:RegisterBarClass("ActionBar", "ActionBar", L["Action Bar"], "Action Button", DB.ActionBar, Neuron.ActionButton, 250)
 
 	--Neuron Bag Bar
-	Neuron:RegisterBarClass("BagBar", "BagBar", L["Bag Bar"], "Bag Button", DB.BagBar, Neuron.BAGBTN, Neuron.NUM_BAG_BUTTONS) --Neuron.NUM_BAG_BUTTONS == 5 for retail and 6 for classic due to the keyring
+	Neuron:RegisterBarClass("BagBar", "BagBar", L["Bag Bar"], "Bag Button", DB.BagBar, Neuron.BagButton, Neuron.NUM_BAG_BUTTONS)
```

Comment:
Nice! Button spelled out without abbreviations and everything. Code is for reading not writing :-)